tests/e2e/operator.sh

```diff
@@ -266,7 +265,7 @@ wait_for_stabilization() {
 	count=0
 	while true; do
 		change=0
-		pod_info=$(kubectl get pods -n confidential-containers-system -o=jsonpath='{range .items[*]}{.metadata.name}{" "}{range .status.containerStatuses[*]}{.name}{" "}{.restartCount}{"\n"}{end}{end}')
+		pod_info=$(kubectl get pods -n $op_ns -o=jsonpath='{range .items[*]}{.metadata.name}{" "}{range .status.containerStatuses[*]}{.name}{" "}{.restartCount}{"\n"}{end}{end}')
```

Comment:
To be sure it'd be better to quotate the `$op_ns` (just in case a space/whatever-weird-char is ever added)